Sources/Web3Core/KeystoreManager/EtherscanTransactionChecker.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  EtherscanTransactionChecker.swift
+//  Created by albertopeam on 28/12/22.
+//
+
+import Foundation
+import _Concurrency
+
+public struct EtherscanTransactionChecker: TransactionChecker {
```

Comment:
It would be nice to think if we want to have more than the Etherscan implementation.
To be clear, I haven't checked other providers.

Reply:
I guess it could be done in future, I mean of course we do, since there's a bunch of networks that almost as much popular as Mainet, but we have to start somewhere.